Week1/FAndresVera/test.txt

```diff
@@ -0,0 +1,8 @@
+This is a text file used to manage the conflicts between branches in a repository of Github. 
+
+<<<<<<< HEAD
```

Comment:
This is a bad resolution

<<<<<<< HEAD

>>>>>>> conflictBranch

Shouldn't be here

If this is a Java file, it won't compile